frontend/src/components/ProjectCard/index.tsx

```diff
@@ -20,19 +34,17 @@ const ProjectCard = ({ projectName, summary, ownerName, repoName }) => {
       href={`/${ownerName}/${repoName}${isSingleProject ? '' : '/' + projectName}`}
     >
       <Card as="a" className={styles.card} data-cy="project-card">
-        <div className={styles.thumbnail}>
-          <div>
-            {isLoading || isError ? null : (
-              <Image
-                alt={`Render of PCB of ${nameOnCard}`}
-                data-cy="project-card-thumbnail"
-                height={180}
-                objectFit="contain"
-                src={src}
-                width={240}
-              />
-            )}
-          </div>
+        <div ref={intersectionObserverRef} className={styles.thumbnail}>
+          {isLoading || isError ? null : (
+            <Image
+              alt={`Render of PCB of ${nameOnCard}`}
+              data-cy="project-card-thumbnail"
+              height={180}
+              objectFit="contain"
```

Comment:
```suggestion
              objectFit="contain"
              priority={true}
```

At this stage the HTML is loaded, this image is present and in view so it makes sense to load it as quickly as possible.

Reply:
I handled this in #515, It's [advisable](https://nextjs.org/docs/api-reference/next/image#priority) to set the priority to `true` for the images that appear in [LCP](https://web.dev/lcp/) in the initial render.

Reply:
I read that advice too but I'm not sure it applies here. The card data is getting loaded lazily so shouldn't it be guaranteed to be "in view" when rendered? If so then why the additional lazy loading? We want to see the image now!

In practice it's likely imperceptible but we might as well make it make sense? #515 it looks more complicated, passing in the `priority` as a prop to the card. What do we gain by doing that?

Reply:
Hmmm... alright then.